src/test/java/sk/freemap/gpxAnimator/ui/TestDurationSpinnerModel.java

```diff
@@ -1,25 +1,57 @@
 package sk.freemap.gpxAnimator.ui;
 
-import org.junit.Test;
 
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
 public class TestDurationSpinnerModel {
-  
 
-  @Test
-  public void sampleTest() {
-  	System.out.println("This is a sample Test.");
-  }
+    private static List<Long> edgeCases = new ArrayList<>();
+
+    @BeforeAll
+    public static void Setup(){
+        edgeCases.add(Long.MIN_VALUE);
+        edgeCases.add(Long.MIN_VALUE);
+        edgeCases.add(-1L);
+        edgeCases.add(0L);
+        edgeCases.add(100L);
+        edgeCases.add(null);
+    }
+
+    @Test
+    public void DurationSpinnerModelSetGetTest() {
+
+        DurationSpinnerModel d = new DurationSpinnerModel();
+
+        for (Long edgeCase : edgeCases) {
+            d.setValue(edgeCase);
+            assertEquals(edgeCase, d.getValue());
+        }
```

Comment:
This could be done using parametrized tests. I would like to implement this because I haven't done parametrized tests for a long time... :-)